kuma/static/js/users.js

```diff
@@ -95,9 +107,13 @@
         // Set the new tag element text
         $('#tagit-interests .tagit-new input').attr('placeholder', gettext('New interest...'));
 
-        // Convert the expertise text field into tag list with checkboxes sync'd to
-        // interests
-        $("#id_user-expertise").hide().after("<ul id='tags-expertise' class='tags'></ul>");
+        // Convert the expertise text field into tag list
+        // lowercase to deal with database weirdness
+        // checkboxes sync'd to interests
+        var expertise = $('#id_user-expertise').val().toLowerCase();
+        $('#id_user-expertise').hide()
+        .val(expertise)
+        .after("<ul id='tags-expertise' class='tags'></ul>");
```

Comment:
Nit: For consistency, should this not be wrapped in single quotes and then double quotes inside?